packages/salesforcedx-utils-vscode/src/constants.ts

```diff
@@ -5,4 +5,6 @@
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+export const CLIENT_ID = 'sfdx-vscode';
+export const DEFAULT_CONNECTION_TIMEOUT_MS = 20000;
```

Comment:
You moved this here but I don't see a removal. Where was this previously?

Reply:
Removed. It'll be updated with this next push. I'm just waiting on comments from Jonathan so I don't needlessly trigger appveyor runs.

Reply:
Pushing the change up with that in it now.